test/isolation/specs/deadlock_dropchunks_select.spec

```diff
@@ -17,7 +17,7 @@ teardown
 
 session "s1"
 setup	{ BEGIN; SET TRANSACTION ISOLATION LEVEL READ COMMITTED; SET LOCAL lock_timeout = '50ms'; SET LOCAL deadlock_timeout = '10ms'; }
-step "s1a"	{ select drop_chunks( '2018-12-25 00:00'::timestamptz, 'dt'); }
+step "s1a"	{ select show_chunks(hypertable => 'dt', older_than => '2018-12-25 00:00'::timestamptz); select drop_chunks( '2018-12-25 00:00'::timestamptz, 'dt'); }
```

Comment:
The changes to this spec are not necessary. Could you revert them?

Reply:
Done.